src/client.ts

```diff
@@ -389,6 +389,41 @@ export class GenesysCloudWebrtcSdk extends (EventEmitter as { new(): StrictEvent
     }
   }
 
+  /**
+   * Update the default media settings from the config.
+   *
+   * If `updateActiveSessions` is `true`, any active sessions will
+   *  have their outgoing media devices updated.
+   *
+   * Else, only the defaults will be updated and active sessions'
+   * media devices will not be touched.
+   *
+   * @returns a promise that fullfils once the default
+   *  settings and sessions are updated (if specified)
+   */
+  async updateDefaultMediaSettings (settings: IMediaSettings & { updateActiveSessions?: boolean }): Promise<any> {
+    const allowedSettings: Array<keyof IMediaSettings> = [
+      'micAutoGainControl',
+      'micEchoCancellation',
+      'micNoiseSuppression',
+      'monitorMicVolume'
+    ];
+
+    const entries = (Object.entries(settings) as Array<[keyof IMediaSettings, any]>)
+      .filter(([setting]) => allowedSettings.includes(setting));
+
+    this.logger.info('updating media settings', entries);
```

Comment:
Should we log `updateActiveSessions` along with `entries`?

Reply:
nah, I just wanted to see the settings. there are other logs in the update fn.